docs/adr/ADR-6_dynamic-config.md

```diff
@@ -0,0 +1,39 @@
+# ADR 6 Dynamic Configuration
+
+## Status
+
+Proposed
+
+## Context
+
+The configuration of validators are mounted into Connaisseur as a configmap, as it it common practice in the Kubernetes ecosystem. When this configmap is upgraded, say with a `helm upgrade`, the resource itself in Kubernetes is updated accordingly, but that doesn't mean it's automatically updated inside the pods which mounted it. That only occurs once the pods are restarted and util they aren't the pods still have an old version of the configuration lingering around. This is a fairly unintuitive behavior and the reason why Connaisseur doesn't mount the image policy into its pods. Instead, the pods have access to the kube API and get the image policy dynamically from there. The same could be done for the validator configuration, but there is also another more kubernetes native solution.
```

Comment:
`but there is also another more kubernetes native solution.` already sounds like an evaluation and therefore shouldn't be in the intro, I'd argue